### PR TITLE
create articles page hbs related to #15

### PR DESCRIPTION
**Pull request by SalwaMugh**
converting the html file to hbs .
testing the /articles route.

### CODE REVIEW

File: src/controllers/articles.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+exports.get = (req, res) => {
+  res.render('articles', {
+    title: 'Articles ', headerFound: true, footerFound: true, style: ['articles', 'header', 'footer'], javascript: ['hambruger'],
```
Comment (knanahassouna1) [inline src/controllers/articles.js]: i think you need article.js file , and you don't import it in javascript array.

File: test/test_BE.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  supertest(app)
+    .get('/articles')
+    .expect(200)
+    .end((err, res) => {
+      if (err) {
+        t.error(err);
+      }
+      t.ok(res.text.includes('body'));
+      t.equal(res.text.substr(0, 15), '<!DOCTYPE html>', 'the respone  should be html file');
+      console.log(res.header);
```
Comment (knanahassouna1) [inline test/test_BE.js]: remove console.log

File: public/css/articles.css [hunk 3]
```diff
@@ -0,0 +1,125 @@
+/* @import url('https://fonts.googleapis.com/css?family=Lato'); */
+
+@import url('https://fonts.googleapis.com/css?family=Lato:300,400');
+@import url('https://fonts.googleapis.com/css?family=Cairo');
+* {
```
Comment (knanahassouna1) [inline public/css/articles.css]: remove ```*{padding: 0;
  margin: 0;
  font-family: lato, sans-serif;
}```

File: public/css/articles.css [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+  padding: 0;
+  margin: 0;
+  font-family: lato, sans-serif;
+}
```
Comment (mohannadhanafi) [inline public/css/articles.css]: Just link public.css instead of repeating this code

File: src/controllers/articles.js (hunk 1 above)
Comment (SalwaMugh) [reply src/controllers/articles.js]: I still didn't use it so I didn't require it here , when I need it I would put it :smiley_cat:

File: test/test_BE.js (hunk 2 above)
Comment (SalwaMugh) [reply test/test_BE.js]: done

File: public/css/articles.css (hunk 3 above)
Comment (SalwaMugh) [reply public/css/articles.css]: done

**Comment by SalwaMugh** [comment]
I will do the tests as soon as I finish the backend tasks :smile:

**Pull request closed without merging**